src/app/actions.ts

```diff
@@ -34,12 +23,40 @@ const fuseOptions = {
   includeScore: true,
   keys: ["name"],
 };
-const fuse = new Fuse<FlattenedRegion>(flattenedRegionCodes, fuseOptions);
+const regionsFuseIndex = new Fuse<FlattenedRegion>(
+  flattenedRegionCodes,
+  fuseOptions,
+);
 
 export async function findMatchingRegions(query: string) {
-  const regions = fuse.search(query);
+  const regions = regionsFuseIndex.search(query);
   if (!regions || regions.length === 0) {
     throw new Error(`Could not find any regions matching ${query}`);
   }
   return regions.slice(0, 10);
 }
+
+export type Species = {
+  sciName: string;
+  comName: string;
+  speciesCode: string;
+  category: string;
+  taxonOrder: number;
+  bandingCodes: string[];
+  comNameCodes: string[];
+  sciNameCodes: string[];
+  order: string;
+  familyCode: string;
+  familyComName: string;
+  familySciName: string;
+};
+
+export type SpeciesGetResponse = FuseResult<Species>[];
+
+const speciesFuseIndex = new Fuse<Species>(speciesData as readonly Species[], {
+  keys: ["comName"],
+  ignoreLocation: true,
+});
+export async function findSpecies(query: string): Promise<SpeciesGetResponse> {
+  return speciesFuseIndex.search(query).slice(0, 10);
+}
```

Comment:
These functions are pretty small and chill thanks to fuse handling most of the internals. At some point this file will become large enough that it might make sense to move each server action into its own file inside a folder of them. 

e.g 
```bash
actions/
├─ regionSearch.ts
├─ speciesSearch.ts
```